src/content/docs/apm/agents/php-agent/php-agent-api/newrelic_set_error_group_callback.mdx

```diff
@@ -0,0 +1,225 @@
+      <th width="25%">
+        API Parameter
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$callback` (_function_)
+      </td>
+
+      <td>
+        Required. Provide a callback function that will be registered with the PHP Agent.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Callback Parameters
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$txndata` (_array_)
+      </td>
+
+      <td>
+        Required. An array of transaction data provided to your callback by the PHP Agent.
+      </td>
+    </tr>
+    <tr>
+     <td>
+       `$errdata` (_array_)
+     </td>
+     <td>
+       Required. An array of error data provided to your callback by the PHP Agent.
+     </td>
+    </tr>
+  </tbody>
+</table>
+
+### Array Key/Value pairs
+
+#### `$txndata` - PHP Agent transaction data provided to your callback
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Key
+      </th>
+      <th>
+        Value
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        "request_uri"
+      </td>
+      <td>
+        (_string_) Request URI
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "path"
+      </td>
+      <td>
+        (_string_) Filepath
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "method"
+      </td>
+      <td>
+        (_string_) HTTP method (`GET`, `POST`, etc.)
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "status_code"
+      </td>
+      <td>
+        (_int_) HTTP status code (`200`, `404`, etc.)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+#### `$errdata` - PHP Agent error data provided to your callback
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Key
+      </th>
+      <th>
+        Value
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        "klass"
+      </td>
+      <td>
+        (_string_) Class name
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "message"
+      </td>
+      <td>
+        (_string_) Error message
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "file"
+      </td>
+      <td>
+        (_string_) Filepath
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "stack"
+      </td>
+      <td>
+        (_string_) JSON error strack trace
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<Callout variant="important">
+  - Not all keys are guaranteed to contain values. This is highly dependent on the user application and the nature of the error.
+  
+  - Array keys will always be set regardless of whether or not they contain empty values.
+</Callout>
+
+## Return values
+
+API returns `true` if the callback is registered successfully, otherwise `false`.
+
+## Examples
+
+```php
+$callback = function(array $txndata, array $errdata) 
+{
+    // custom code to parse array data
```

Comment:
Do we want to add a simple if/else to optionally set the fingerprint based on one of the data? This example would be more useful then, in my opinion.

Reply:
Made improvements in 4b8023d3a9fc8f754b2bb47dfa5f1086559bf3a7